requirements/test.in

```diff
@@ -2,11 +2,12 @@
 
 -r base.txt                         # Package dependencies
 
--e git+https://github.com/edx/xblock-sdk.git@446e979bad1f3bf7f891787b32d5b2e7620b2f23#egg=xblock-sdk==0.1.9
+bok_choy==0.7.1
```

Comment:
Why did bok-choy and selenium need to be pinned?  And if they do, shouldn't that go into a constraints file with an explanation?

Reply:
Done.